gst/edge/edge_sink.c

```diff
@@ -287,9 +304,17 @@ gst_edgesink_start (GstBaseSink * basesink)
   int ret;
   char *port = NULL;
 
-  ret =
-      nns_edge_create_handle (NULL, self->connect_type,
-      NNS_EDGE_NODE_TYPE_PUB, &self->edge_h);
+  if (NNS_EDGE_CONNECT_TYPE_CUSTOM != self->connect_type) {
+    ret = nns_edge_create_handle (NULL, self->connect_type,
+        NNS_EDGE_NODE_TYPE_PUB, &self->edge_h);
+  } else {
+    if (!self->custom_lib) {
+      nns_loge ("Failed to start edgesink. Custom library is not set.");
+      return FALSE;
+    }
+    ret = nns_edge_custom_create_handle (NULL, self->custom_lib,
+        NNS_EDGE_NODE_TYPE_PUB, &self->edge_h);
+  }
```

Comment:
Thank you for the great PR :+1: 
If the value of ```self->custom_lib``` is always ```NULL``` or contains ```a valid pointer value```, we can simply modify the code as follows. How do you think? :smile: 
```
  if (self->connect_type == NNS_EDGE_CONNECT_TYPE_CUSTOM && self->custom_lib) {
    ret = nns_edge_custom_create_handle (NULL, self->custom_lib, NNS_EDGE_NODE_TYPE_PUB, &self->edge_h);
  } else {
    ret = nns_edge_create_handle (NULL, self->connect_type, NNS_EDGE_NODE_TYPE_PUB, &self->edge_h);
}
 ```